//
// Created by 刘优 on 16/6/3.
//
syntax = "proto3";

package webmonitor;

message HTTP_CONTENT {
    enum METHOD {
      UNKNOWN = 0;
      HEAD = 1;
      GET = 2;
      POST = 3;
    }
    enum MATCHCMD {
      NOT_CARE = 0;
      YES = 1;
      NO = 2;
    }
    METHOD method = 1;
    MATCHCMD match_cmd = 2;
    string match_content = 3;
}

message PING_CONTENT {
    int32 pkg_num = 1;
    int32 pkg_size = 2;
}

message DNS_CONTENT {
    string dns = 1;
}
